pyiron_atomistics/atomistics/structure/atoms.py

```diff
@@ -196,6 +196,31 @@ def __init__(
         self._visualize = Visualize(self)
         self._analyse = Analyse(self)
 
+    @property
+    def spins(self):
+        """
+        Magnetic spins for each atom in the structure
+
+        Returns:
+            numpy.ndarray/list: The magnetic moments for reach atom as a single value or a vector (non-collinear spins)
+
+        """
+        if self.has("initial_magmoms"):
+            return self.get_array("initial_magmoms")
+        else:
+            return
```

Comment:
Unless this is how we do it already with other attributes, I think this should be an `AttributeError`.

Reply:
If we have to be compatible to ase, we shouldn't raise an `AttributeError` and should simple return `None`.

Reply:
Ah, bummer. :+1: